core/src/main/scala/kafka/utils/json/DecodeJson.scala

```diff
@@ -17,10 +17,8 @@
 
 package kafka.utils.json
 
-import scala.collection.{Map, Seq}
-import scala.collection.compat._
+import scala.collection.{Factory, Map, Seq}
```

Comment:
please revert this unrelated changes

Reply:
We are getting rid of `scala.collection.compat._` since it was a workaround package for cross-building with Scala 2.12. Now, we can import Factory directly from `scala.collection`.